### ISSUE TITLE
[Feature] More diverse blood sources

### ISSUE THREAD

**Issue by Jackiecrazy** [TeamLapen/Vampirism#886]
## What would you like to have changed or added to Vampirism

A config option to make vampires capable of going hungry, in addition to the thirst requirement. Lore-wise, it could be a "cursed" mode where your body still needs to eat to stay alive, but cannot replenish lost blood or something.

## Does this make the mod experience more balanced?
It does not, but I want to be miserable. It also helps with compat for food mods like spice of life and harder natural healing, which change the food-health dynamic.

## Can you provide additional resources (etc. textures or models)
Probably won't be necessary.

**Comment by Sadzony** [comment]
Spice of life still works, allowing you to eat food to balance your diet and gain the diet bonuses even as a vampire! So that makes vampires not need food for sustenance and for vitamins instead, and I  feel like that is good enough :D I am by no means against your idea, just putting my opinion out there

**Comment by MarioSMB** [comment]
I think this would be a nice improvement, maybe one could take it a step further; thirst no longer *replaces* hunger at all but instead accounts for the extra powers vampires have (flight, regeneration etc), while normal human-like behaviour would still be governed by food, making regular farms still relevant and keeping players from just stocking up on hearts late-game.

**Comment by maxanier** [comment]
Making blood a dedicated resources used for skills would be a fundamental change of the blood system (more towards something like BloodMagic uses afaik). I don't think it would be a bad system, maybe even better, but definitively different. I don't think I want such a change at the moment

**Comment by MarioSMB** [comment]
That's fair, I have been thinking about how blood magic could be incorporated into the mod to effectively transform the progression into something more expansive and engaging in gameplay; elements of it already exist in the form of blood spikes and rituals.  
But understandably there is only so much one can do and such a large change can shake things up for the worse if one is not careful!

**Comment by kaboom24** [comment]
Not a big fan. I feel like this takes away one of the core vampire elements of only being able to eat blood. If it gets added at least make it optional.

**Comment by MarioSMB** [comment]
I agree that blood should remain the sole source of nutrition for vampires, my aforementioned concerns are more associated with the stagnation of late-game vampirism food sources; you can just buy out an entire stack of hearts and that's all you ever need for sustenance.  

For that reason it does seem like a shame to cut off the vampirism playthrough from needing farms and such, but if animals were to become a more staple food source for vampires (and possibly the villagers that look after them), vampires would still have *some* use for crop and animal farms, if not necessarily to eat themselves.  
So I think the focus should be on fleshing out the (drink) sources for vampires into the mid-to-late game rather than trying to reintegrate the hunger mechanic.

**Comment by kaboom24** [comment]
yeah i like that idea, I do think that abilities should consume a little
blood. as long as blood is the only food source, I do think that we should
flesh out things such as eating villagers and farming. Perhaps make
plants/creatures that can be grown for blood and make hearts harder to
obtain by removing them from trades since I dont think even a vampire
villager would be capable of taking a humans heart?
